README.md

```diff
@@ -12,3 +12,23 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the span of the parallel program, in terms of worst-case $\Theta$? Hint:
 It may help to consider the DAG of the parallel program.
+
+## Runtime Anaylsis, Maxie M.
+### Recursive Mergesort 
+- will recursively divide array and will perform merge operation
+- each recusive call to sort the left and right halves can in parallel
+  - Although merging of these halves must happen sequentially
+- **depth of recursion at each level:** $log(n)$
+- **merge operation processes:** $\Theta(n)$
+- **span of the parallel program:** $\Theta(n)$
+### Iterative Mergesort 
+- splits the array into sub-arrays and performs merges in parallel within each iteration
+- **span of the parallel program:** $\Theta(n)$
+### Conclusion 
+- The *spam of the parallel program*, in the terms of *worst-case $\Theta$* is $\Theta(n)$
```

Comment:
At the end, what would be the total time complexity? O(n) is not a correct answer.